src/utils/MessageBuilder.ts

```diff
@@ -97,6 +100,59 @@ export default class MessageBuilder {
     };
   }
 
+  static reportedTeamButtons(buttonInteraction: ButtonInteraction, activeMatchEmbed: MessageEmbed): MessageOptions {
+    const primaryStyle = "PRIMARY";
```

Comment:
Do we need this to be its own variable? I feel like it'd be simpler just to use the string instead of this variable.

If we want to stick with the variable, can we add the appropriate type to help enforce the possible values?

Reply:
Resolved. - Turned to string.